kfs-core/src/test/java/org/kuali/kfs/sys/service/ReportGenerationServiceTest.java

```diff
@@ -78,6 +78,7 @@ public void testGenerateReportToPdfFile_WithDataSource() throws Exception {
             reportGenerationService.generateReportToPdfFile(reportData, accountDataSource, template, fullReportFileName);
         }
         catch (Exception e) {
+            e.printStackTrace();

```

Comment:
we probably want logging here (and below) instead of just printing the stacktrace, right?